### ISSUE TITLE
Make sure rich image data structure has a "label" property

### ISSUE THREAD

**Issue by ctrueden** [imagej/imagej-common#47]
ImageJ1 displays the __label__ as a "subtitle" at the top of the image window (_not_ in the title bar—that would be the __name__). ImageJ2 will also need a dedicated label property, since it's useful.

We also need slice labels—but see #21 for that.

**Comment by lannybroo** [comment]
I agree with this. I use these all the time in ImageJ1. I also have always found it strange that this is different metadata than the slice labels. It would be nice if you wanted a subtitle on an entire stack to be able to set a label/subtitle property once for a stack. In IJ1, I end up using a utility function often when I want a plugin to be able to set a subtitle. Here is my utility function, for reference:

    public static void setSubtitle(ImagePlus imp, String label) {
        ImageStack stack = imp.getStack();
        if (stack.getSize()>1) {
            // Stacks show the stack label as the subtitle
            stack.setSliceLabel(label, imp.getSlice());
        } else {
            // Single images show the "Label" property as the subtitle
            imp.setProperty("Label", label);
        }
        imp.updateAndRepaintWindow();
    }

**Comment by ctrueden** [comment]
Thanks @lannybroo. I think that discrepancy is an artifact of ImageJ1's growth from 2D to 3D to 5D over time. Hopefully we can make ImageJ2 more unified in this regard.
